modules/cbMap/processmap/Validations.php

```diff
@@ -324,6 +324,12 @@ public static function formatValidationErrors($errors, $module) {
 				$error.= $err . "\n";
 			}
 		}
+		if (strpos($error, "{custommsg|") > 0) {
+			preg_match_all('/{(.*)}/', $error, $match);
+			$res = explode('|', $match[1][0]);
+			include_once 'include/validation/'.$res[1].'.php';
+			$error = call_user_func(array(__NAMESPACE__ .$res[1], $res[2]), 'email');
```

Comment:
The first "foreach" contains the fieldname that you are validating, you have to use that here, not "email"

```
foreach ($errors as $errs) {
change it to
foreach ($errors as $field => $errs) {
```

then change the hardcoded email for the variable $field